recipe/meta.yaml

```diff
@@ -1,17 +1,14 @@
 {% set version = "2.38.0" %}
-{% set win_version = version[:-2] %}
 
 package:
   name: graphviz
   version: {{ version }}
 
 source:
-  fn:  graphviz-{{ version }}.tar.gz  # [unix]
-  url: http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-{{ version }}.tar.gz  # [unix]
+  url: http://pkgs.fedoraproject.org/repo/pkgs/graphviz/graphviz-2.38.0.tar.gz/5b6a829b2ac94efcd5fa3c223ed6d3ae/graphviz-2.38.0.tar.gz  # [unix]
   md5: 5b6a829b2ac94efcd5fa3c223ed6d3ae  # [unix]
```

Comment:
Note the URL above is just using `md5`. So this can be templated out easily.

Reply:
That is true, but for the next release we will simply have a gitlab URL which only contains the version and not the md5. Hence, I would argue for keeping it like this for now.

Reply:
My concern is the autoupdate bot will come through and create a PR that changes the version up top in the Jinja, but the URL will stay the same. If that passes, someone could end up merging this not realizing the URLs are incorrect. This has happened before. ( https://github.com/conda-forge/sqlite-feedstock/issues/25 ) Hence the request to fix up the URLs. Am ok if it doesn't generalize and does fail on the next release as that will cause someone to look into it instead of potentially overlooking it and merging because the CI passed for something that is incorrect.

Reply:
Ah, that makes sense, I see. Will do that. Suggestion: let the autoupdate bot check whether URL changes after changing the version.